openapi.yaml

```diff
@@ -16,175 +16,301 @@ host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/notes-api
 produces:
   - application/json
 paths:
-  /pets:
+  /notes:
     get:
-      summary: List all pets
+      summary: Get notes for a student
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - notes
+      operationId: getNotes
       parameters:
-        - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
-        - in: query
-          name: species
-          description: Search by pet species
+        - name: studentID
+          in: query
+          type: string
+          required: true
+          description: 9-digit OSU ID of student whose notes will be retrieved
+        - name: q
+          in: query
+          type: string
           required: false
+          description: Query string to be used when searching through the contents of each note
+        - name: sortKey
+          in: query
           type: string
+          required: false
+          default: 'dateModified'
+          description: The key by which to sort the notes in the response
+        - name: sources
+          in: query
+          type: array
+          items:
+            type: string
+            enum: &SOURCES
+              - advisorPortal
+              - banner
+              - degreeWorks
+            collectionFormat: csv
+          required: false
+          description: A comma-separated list of sources to filter by
+        - name: contextTypes
+          in: query
+          type: array
+          items:
+            type: string
+            collectionFormat: csv
+          required: false
+          description: A comma-separated list of context types to filter by
       responses:
         200:
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/NotesResult'
         400:
-          description: Bad Request
+          description: Bad request
           schema:
             $ref: '#/definitions/Errors'
         500:
-          description: Internal Server Error
+          description: Internal server error
```

Comment:
Good catch. Can you also fix it in this [PR](https://github.com/osu-mist/express-api-skeleton/pull/15)?

Reply:
Sure, should be fixed now